docs/devGuide/maintainerGuide.md

```diff
@@ -112,7 +120,27 @@ $ git commit -m 'Update vue-strap version to v2.0.1-markbind.XYZ'
 
 * **Note:** We uniquely do this for each MarkBind release (rather than spontaneously update the vue-strap files for each affected PR), in order to reduce unnecessary merge conflicts. It also makes it easier for the maintainers to vet the changes.
 
-2. Rebuild the test files.
+2. Do an `npm version` to increment the version number. Which to increment (`patch`, `minor` or `major`) depends on what PRs are merged for the new version, which means you must know beforehand about the changes.
```

Comment:
Do ~an~ -> a `npm version` ...

Reply:
Actually, "an npm version" is correct. As with "a user".
https://ell.stackexchange.com/questions/26986/why-a-user-instead-of-an-user